decidim-dev/lib/decidim/dev/test/rspec_support/capybara_select2.rb

```diff
@@ -10,7 +10,10 @@ def select2(value, from:)
       select2_container.find(".select2-selection").click
 
       body = find(:xpath, "//body")
-      expect(body).to have_selector(".select2-dropdown li.select2-results__option", text: value)
+
+      # `wait: 1_000_000` is a temporary test to see if the culprit for all the
+      # failing tests is that our scope filtering is too slow
+      expect(body).to have_selector(".select2-dropdown li.select2-results__option", text: value, wait: 1_000_000)
```

Comment:
Should we keep this in `master`?

Reply:
No, please! :) The proposal in https://github.com/decidim/decidim/issues/2367#issuecomment-352195505 is better, I'll remove it!